### PR TITLE
Renamed property splitSize to numSplits (support backward compatibility)

### PR DESCRIPTION
**Pull request by pravinbhat**
Renamed property splitSize to numSplits while supporting backward compatibility
Updated Readme

### CODE REVIEW

File: src/main/scala/datastax/astra/migrate/AbstractJob.scala [hunk 1]
```diff
@@ -7,7 +7,7 @@ class AbstractJob extends BaseJob {
 
   abstractLogger.info("PARAM -- Min Partition: " + minPartition)
   abstractLogger.info("PARAM -- Max Partition: " + maxPartition)
-  abstractLogger.info("PARAM -- Split Size: " + splitSize)
+  abstractLogger.info("PARAM -- Number if Splits : " + numSplits)
```
Comment (msmygit) [inline src/main/scala/datastax/astra/migrate/AbstractJob.scala:10]:
```suggestion
  abstractLogger.info("PARAM -- Number of Splits: " + numSplits)
```

File: src/main/scala/datastax/astra/migrate/BaseJob.scala [hunk 2]
```diff
@@ -49,7 +49,8 @@ class BaseJob extends App {
   val minPartition = new BigInteger(Util.getSparkPropOr(sc, "spark.origin.minPartition", "-9223372036854775808"))
   val maxPartition = new BigInteger(Util.getSparkPropOr(sc, "spark.origin.maxPartition", "9223372036854775807"))
   val coveragePercent = Util.getSparkPropOr(sc, "spark.coveragePercent", "100")
-  val splitSize = Integer.parseInt(Util.getSparkPropOr(sc, "spark.splitSize", "10000"))
+  val splitSizeBackwardCompatibility = Util.getSparkPropOr(sc, "spark.splitSize", "10000")
```
Comment (msmygit) [inline src/main/scala/datastax/astra/migrate/BaseJob.scala:52]: I don't see `splitSizeBackwardCompatibility` being used anywhere. What am I missing?

**Review by msmygit — APPROVED** [review]
LGTM 👍🏼

File: src/main/scala/datastax/astra/migrate/AbstractJob.scala (hunk 1 above)
Comment (pravinbhat) [reply src/main/scala/datastax/astra/migrate/AbstractJob.scala]: @msmygit Fixed

File: pom.xml [hunk 3]
```diff
@@ -8,7 +8,7 @@
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <revision>3.2.1</revision>
+    <revision>3.2.2</revision>
```
[the remark this answers is not in the document]
Comment (pravinbhat) [reply pom.xml:11]: @msmygit Cool, when does it do the actual bump? On merge to main?

[the remark this answers is not in the document]
Comment (mieslep) [reply pom.xml:11]: These should generally bump on release, but that's not really resolved as yet... arguably packaging, docker build, etc should also happen on release...

**Pull request merged**